[SDK Docs Audit] Android Integrating the SDK (#9429)
braze-inc/braze-docs: _includes/developer_guide/android/sdk_integration.md

## Integrating the Android SDK

### Add Braze to `build.gradle`

> Add the following code to your `build.gradle`:

Reviewer: @chshapiro , qq: is this `build.gradle` in this line, the same `app/build.gradle` on line 13? or is one in root and one in `./app/` directory?

```
repositories {
  mavenCentral()
}
```

> Next, add the library which you wish to use to your build.gradle():

Reviewer 2: ```suggestion
Next, add the library which you wish to use to your `app/build.gradle`:
```

```
dependencies {

>     implementation 'com.braze:android-sdk-base:CURRENT_SDK_VERSION'

Reviewer: @chshapiro , does `CURRENT_SDK_VERSION` mean the developers current version? or the latest version for the Android sdk as specified [here](https://www.braze.com/docs/developer_guide/changelogs/?sdktab=android)?

## Integrating the Android SDK

### Step 1: Update your `build.gradle`

> In your `build.gradle`, add `mavenCentral()` to your list of repositories.

Reviewer: Can i ask what this is for? as a reader (who admittedly nows little about android stuff) i'm curious why i'm adding a repo that doesn't have the word "braze" in it. something like:

This is a repository for ... (something very simple).

Author: It's a repository used to bring in third-party libraries. It's pretty self explanatory for Android developers.

> In your `build.gradle`, add `mavenCentral()` to your list of repositories.

Reviewer: ```suggestion
In your `build.gradle`, add [`mavenCentral()`](https://docs.gradle.org/current/kotlin-dsl/gradle/org.gradle.api.artifacts.dsl/-repository-handler/maven-central.html) to your list of repositories.
```

```kotlin
repositories {
  mavenCentral()
}
```

Next, add Braze to your dependencies.

{% tabs local %}
{% tab base only %}

> If you don't plan on using Braze UI components, add the following code to your `build.gradle`. Replace `CURRENT_SDK_VERSION` with the THIS:

Reviewer: i left a placeholder here (`THIS`), related to the question i asked previously. but to ensure it's not lost: is this the developers current SDK version or the Android Braze SDK's _latest_ version?

Author: It's the version they will be using which should ideally be the latest Android SDK version from the changelog

See our SDK reference documentation for more information on the parameters available for [`BrazeActivityLifecycleCallbackListener`](https://braze-inc.github.io/braze-android-sdk/kdoc/braze-android-sdk/com.braze/-braze-activity-lifecycle-callback-listener/index.html).

## Optional configurations

> ### Location tracking

Reviewer 2: Move "Location Tracking" below "Test session tracking".  Otherwise, we have the Test step being under the "Optional configurations" heading.

Author: the testing is optional - it's specifically marked as optional but i moved everything around

Starting with Braze Android SDK version 3.6.0, Braze location collection is disabled by default.
{% endalert %}

> ### Test session tracking

Reviewer: If this is supposed to be nested under the original then you'll want to keep `Step 5:` in the heading, however, if you are trying to move it to its own H2 heading, then you'll want to do this:

```suggestion
## Testing session tracking
```

Either way works, feel free to choose the option you think is best 👍🏽  @torossian97

Author: It should be under optional configurations

> Replace `MIN_LOG_LEVEL` with the **Value** of the log level you'd like to set as your minimum log level. Any logs at a level `>=` to your set `MIN_LOG_LEVEL` will be forwarded to Android's default [`Log`](https://developer.android.com/reference/android/util/Log) method. Any logs `<` your set `MIN_LOG_LEVEL` will be discarded.

Reviewer 2: Right here we're referencing the "Value" column, so we can't remove it.

Author: reverted

#### Suppressing logs

> To suppress all logs for the Braze Android SDK, call `BrazeLogger.SUPPRESS` in your application's `onCreate()` method _before_ any other methods.

Reviewer 2: ```suggestion
To suppress all logs for the Braze Android SDK, set the log level to `BrazeLogger.SUPPRESS` in your application's `onCreate()` method _before_ any other methods.
```
